creator_project/packages/creator-luacpp-support/panels/setup-project-panel.html

```diff
@@ -21,10 +21,13 @@ <h2>Setup Target Project <span class="version">v{{version}}</span></h2>
         </div>
     </ui-prop>
 
-    <ui-prop name="Export Resource Only">
+    <ui-prop name="Export Resource">
         <ui-checkbox class="flex-1" v-value="profileProject.data.exportResourceOnly" v-on:confirm="_onChangeExportResourceOnly">
             Export Resource Only
         </ui-checkbox>
+        <ui-checkbox class="flex-1" v-value="profileProject.data.exportDynamicallyLoadResource" v-on:confirm="_onChangeexportDynamicallyLoadResource">
+            Export Dynamically Load Resource
```

Comment:
`Export Dynamically Load Resource` the meaning is not clear, you can ask Jason for some help.

Reply:
`Export Resource Dynamically Loaded` is a better choice, keep the same name style with `Export Resource Only`